CS.API/Program.cs

```diff
@@ -151,17 +152,17 @@
 var app = builder.Build();
 
 if (args.Length == 1 && args[0].ToLower() == "seeddata")
+{
     SeedData(app);
+}
 
 void SeedData(IHost app)
 {
-    var scopedFactory = app.Services.GetService<IServiceScopeFactory>();
+    var scopedFactory = app.Services.GetRequiredService<IServiceScopeFactory>();
```

Comment:
in contrary to Security project, you were using GetService instead of GetRequiredService. It returns either instance of service or NULL, so it's either you check it for the null reference, or change to GetRequiredService